openslides_backend/action/actions/meeting/import_.py

```diff
@@ -55,6 +68,13 @@ def perform(
         self.index = 0
         action_data = self.get_updated_instances(action_data)
         instance = next(iter(action_data))
+
+        if self.name == "meeting.import":
```

Comment:
I'd rather move this into one method which is called every time and can be overriden to do nothing in the subclass

Reply:
done